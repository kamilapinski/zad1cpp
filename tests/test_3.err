ERROR 2
ERROR 3
ERROR 4
ERROR 5
ERROR 6
ERROR 7
ERROR 9
ERROR 13
ERROR 15
ERROR 17
ERROR 19
ERROR 22
ERROR 25
ERROR 26
ERROR 27
ERROR 29
ERROR 31
ERROR 32
ERROR 33
ERROR 40
