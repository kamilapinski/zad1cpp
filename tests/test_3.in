Japan 2
America
Brazil 4
-Japan 0
-Japan 4
Germany 13241243124
russia 15
Russia 2
-Russia 1
Russia 1
-Russia 1
Japan 2
America 4
United Kingdom 0
United Kingdom  5
=1 2 3
  America 2
America 1
United Kingdom 4
United kingdom 3
Republic    of Kongo 3
france 2
France 2
=999999 1 999999
France 24
France 31
France 12
France 3
France -1
France 0
France  0
 France 1
France 2 
United  Kingdom 2
United  Kingdom 3
United  Kingdom 0
United   Kingdom 2
=1 1 1
=1 2 3
 =3 2 1
=2134 42 32424
=1 1 1