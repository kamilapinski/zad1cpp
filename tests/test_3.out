1. Japan
2. Russia
3. United Kingdom
1. America
1. Republic    of Kongo
1. United kingdom
4. Japan
5. France
5. Russia
7. United Kingdom
1. France
1. Japan
1. United  Kingdom
4. America
4. Republic    of Kongo
4. Russia
4. United   Kingdom
4. United kingdom
9. United Kingdom
1. France
1. United  Kingdom
3. Japan
4. Republic    of Kongo
4. United kingdom
6. Russia
6. United   Kingdom
8. America
9. United Kingdom
1. France
1. United  Kingdom
3. Republic    of Kongo
3. United kingdom
5. America
6. Japan
7. Russia
7. United   Kingdom
9. United Kingdom
1. France
1. Japan
1. United  Kingdom
4. America
4. Republic    of Kongo
4. Russia
4. United   Kingdom
4. United kingdom
9. United Kingdom
